i18n/en.json

```diff
@@ -5213,7 +5213,7 @@
   },
   {
     "id": "store.sql.schema_version.critical",
-    "translation": "The database schema version of %v cannot be upgraded.  You must not skip a version."
+    "translation": "The database schema version of %v is not supported. This Mattermost server supports schema versions 3.0.0 through %v."
```

Comment:
Not sure if this is the best message. The 3.0.0 should be templated to avoid having translation work if we stop supporting old versions. I think It should say "server supports upgrading from schema versions X though X. Downgrades are not supported. Upgrade to X before upgrading to this version." I think that's more clear.